# Define the goal
float32[] x_position
float32[] y_position
float32[] velocity_magnitude
---
# Define the result
float32 x
float32 y
---
# Define a feedback message
float32 x
float32 y
float32 x_velocity
float32 y_velocity
